crates/aptos-crypto/src/unit_tests/hash_test.rs

```diff
@@ -246,3 +248,64 @@ proptest! {
         prop_assert!(HashValue::from_bit_iter(bits2.into_iter()).is_err());
     }
 }
+
+/// Sha3 Buffer overflow tests: https://mouha.be/sha-3-buffer-overflow/
+#[test]
+fn test_sha3_overflow_v256_keccack() {
+    let buffer = b"\x00".repeat(4294967295);
+    let mut sha3 = Sha3::v256();
+    sha3.update(&buffer);
+
+    assert!(true);
+
+    let buffer = b"\x00".repeat(4294967296);
```

Comment:
Ehhh, this will allocate 4 GiB of RAM, so if we merge this, it should be `#[ignore]`, no?